MCprep_addon/spawner/mobs.py

```diff
@@ -444,19 +449,17 @@ def execute(self, context):
 						os.mkdir(dst)
 					except OSError as exc:
 						if exc.errno == errno.EACCES:
-							print("Permission denied, try running blender as admin")
```

Comment:
I'm curious why this was removed? I would reword the print as "Permission denied, verify that you have sufficient permissions to access this file" (since running Blender as admin/root is a terrible idea), but removing this just doesn't make sense.